readme.adoc

```diff
@@ -459,59 +457,57 @@ In the Prometheus dashboard, add the following
 
 [source,bash]
 ----
-round(increase(istio_recommendation_request_count{destination="recommendation.tutorial.svc.cluster.local" }[60m]))
+round(increase(istio_request_count{destination_service="recommendation.tutorial.svc.cluster.local" }[60m]))
 ----
 
-and select Execute
+and select `Execute`
 
 image:readme_images/prometheus_custom_metric.png[alt text]
 
 Then run several requests through the system
 
 [source,bash]
 ----
+# Mac OS: run the following command a few times
 curl customer-tutorial.$(minishift ip).nip.io
+
+# Fedora:
+watch -n 0.5 curl customer-tutorial.$(minishift ip).nip.io
 ----
 
-Note: you may have to refresh the browser for the Prometheus graph to update. And you may wish to make the interval 5m (5 minutes) as seen in the screenshot above.
+NOTE: You may have to refresh the browser for the Prometheus graph to update. And you may wish to make the interval 5m (5 minutes) as seen in the screenshot above.
 
 == Tracing
 
-Tracing in Istio requires to pass a set of headers to outbound requests. It can be
-done manually or using OpenTracing framework instrumentations such as
-https://github.com/opentracing-contrib/java-spring-cloud[opentracing-spring-cloud].
-Framework instrumentation automatically propagates tracing headers and also creates in-process
-spans to better understand what is happening inside the application.
+Distributed Tracing involves propagating the tracing context from service to service, usually done by sending certain incoming HTTP headers downstream to outbound requests. For services embedding an http://opentracing.io/[OpenTracing] framework instrumentations such as https://github.com/opentracing-contrib/java-spring-cloud[opentracing-spring-cloud], this might be a transparent operation. For services that are not embedding OpenTracing libraries, this context propagation needs to be done manually.
```

Comment:
minor changes -
"For services embedding an ..." - remove 'an'.
"this might be a transparent operation" - wasn't sure of using 'operation' - possibly "this might be transparent."?